### DISCUSSION TITLE
怎么更换邮箱？How to change email address?

### DISCUSSION THREAD

**Discussion by z8585** [community/community#141165] [Other Feature Feedback, Questions, & Ideas]
How to change email address?

**Comment by Dhreetiman** [comment]
To change your primary email address on GitHub, you can do the following:

1. Log in to your GitHub account
2. Click your profile picture in the top right corner
3. Click Settings
4. Click Emails in the left sidebar
5. Under Primary email address, click Change email address
6. Follow the prompts to add a new email address, verify it, and set it as the primary email 

You can add as many email addresses to your GitHub account as you like. However, you can't verify email addresses from disposable email address services.

**Reply by z8585** [reply]
Do you have a computer web version?

**Reply by Dhreetiman** [reply]
Yes
